js/bootstrap-slider.js

```diff
@@ -1469,6 +1480,59 @@
 			},
 			_toPercentage: function(value) {
 				return this.options.scale.toPercentage.apply(this, [value]);
+			},
+			_setTicksCSSClass: function(index) {
+				if(typeof this.options.ticks_classes[index] !== 'undefined') {
+					// remove all the classes
+					this._removeClass(this.sliderElem, this.options.ticks_classes.join(" "));
+					// set new classname
+					this._addClass(this.sliderElem,this.options.ticks_classes[index]);
+				}	
+			},
+			_setTicksClass: function() {
```

Comment:
the _setTicksClass looks which tick is the smallest and which is the greatest tick for the selected values.
and then uses the _setTicksCSSClass to set the new css class which should be set for which tick.

the commented code is also just some "test" to get the closest tick. so that the tick css class will be set if the value is closer to one tick than to another tick. but i am not sure if this is usefull at all therefore it is commented.

the code currently have 2 setting possibilities "greatest" and "smallest" but the "closest" option could also be implemented easily.